ci/test-stable.sh

```diff
@@ -1,42 +1,10 @@
 #!/usr/bin/env bash
```

Comment:
How about we use the symlink trick that `gce.sh` and `ec2.sh` use: https://github.com/solana-labs/solana/blob/f0f55af35b3c723f371ace8bc85bc5335586d091/net/gce.sh#L8

test-stable-perf.sh is then a symlink to test-stable.sh, and we select what to do between the pre/post based on the filename and we avoid the test-pre.sh/test-post.sh files

Reply:
Yeah, cool trick, didn't like the pre/post